app/src/main/java/com/engineering/dokkan/view/Favourite/Item_Fragment.java

```diff
@@ -0,0 +1,50 @@
+package com.engineering.dokkan.view.Favourite;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.engineering.dokkan.R;
+
+import java.util.ArrayList;
+
+public class Item_Fragment extends Fragment {
+
+     View v ;
+     private RecyclerView recyclerView;
+     private ArrayList<FavitemModel> data;
+
+    public Item_Fragment() {
+    }
+
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        v = inflater.inflate(R.layout.item_fragment,container,false);
+        recyclerView = (RecyclerView) v.findViewById(R.id.recyclerview_id);
+        ItemRecycAdapter Adapter = new ItemRecycAdapter(data);
+        recyclerView.setLayoutManager(new GridLayoutManager(getActivity(),2));
```

Comment:
recommended to creat initViews method and call it inside onCreateView to keep it simple and clean on